src/configs/lang.ts

```diff
@@ -720,6 +720,11 @@ export const COULD_NOT_LOAD_JURISDICTION_HIERARCHY = translate(
 
 export const STRUCTURES_COUNT = translate('STRUCTURES_COUNT', 'Structures Count');
 
+export const ASSIGNMENT_TYPE = translate('ASSIGNMENT_TYPE', 'Assignment Type');
+
+export const AUTO = translate('AUTO', 'Auto');
```

Comment:
Lets change the test to "Auto-selection"

Reply:
addressed via https://github.com/onaio/reveal-frontend/pull/996/commits/ef166ce296e7f8067317a975812197cbdc973e54